### PR TITLE
style(oui-typography): remove useless mixins

### PR DESCRIPTION
**Pull request by AxelPeter**
* fix margin auto on `hr`
* remove useless mixins

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report yarn-build-ui-kit#1221.0 ✘  
* build
  * build ✘

File: packages/oui-typography/typography.less [hunk 1]
```diff
@@ -14,6 +15,7 @@ abbr[data-oui-tooltip] {
   cursor: help;
 }
 
+// Headings
 h,
```
Comment (antleblanc) [inline packages/oui-typography/typography.less]: I'm not sure that `h` is a valid selector. Am I wrong?

Comment (AxelPeter) [reply packages/oui-typography/typography.less]: Look what's doing the mixin behind :)

**Comment by ovh-cds** [comment]
CDS Report yarn-build-ui-kit#1222.0 ✘  
* build
  * build ✘

File: packages/oui-typography/typography.less (hunk 1 above)
Comment (antleblanc) [reply packages/oui-typography/typography.less]: `&@{index}` alright! Thanks to point out.

**Pull request merged**